views/decode.hbs

```diff
@@ -138,12 +193,12 @@
                         </div>
                         <div class="result-group">
                             <h2>Payload</h2>
-                            <pre>{{ payload }}</pre>
+                            <pre>{{{ payload }}}</pre>
```

Comment:
There is an unnecessary pair of brackets.

Reply:
The brackets are not unnecessary. If you use only two brackets handlebars escapes your input. Thus using two brackets you can't color your output